modules/switch/react/lib/Switch.tsx

```diff
@@ -130,9 +121,17 @@ const SwitchBackground = styled('div')<Pick<SwitchProps, 'checked' | 'disabled'>
     padding: '0px 2px',
     transition: 'background-color 200ms ease',
   },
-  ({checked, disabled}) => ({
-    backgroundColor: disabled ? colors.soap400 : checked ? colors.blueberry500 : colors.licorice200,
-  })
+  ({checked, disabled, theme}) => {
+    if (checked) {
+      return {
+        backgroundColor: disabled ? theme.palette.primary.light : theme.palette.primary.main,
```

Comment:
so we are theming background colors? i guess for the switch it makes sense?

Reply:
Yeah - anything that's blue turns to `theme.palette.primary.main` (other than the focus outlines)